packages/paste-design-tokens/tokens/themes/sendgrid/global/background-color.yml

```diff
@@ -10,6 +10,12 @@ props:
   color-background:
     value: "{!palette-gray-20}"
     comment: Default background color for any container
+  color-background-dark:
```

Comment:
Are these not needed anymore? Seem like left overs from a refactor

Reply:
Nope. Just missed those. I'll delete them.